apps/elasticsearch/templates/services/app/dc.yml.j2

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-kind: DeploymentConfig
+kind: StatefulSet
```

Comment:
Maybe we should create a new type of objects that Arnold should support, but IMO a `StatefulSet` is a kind of deployment...

Reply:
After discussing with @madmatah we decided to create an explicit object type with it's own life cycle or else, these objects would never get cleaned.